extensions/adobe/experience/decisioning/decisioning-asset.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/decisioning-asset",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decisioning Asset",
+  "description": "Decisioning assets are references to assets, which would be used to render the decisionItems. eg) HTML, Image, Json",
+  "type": "object",
+  "meta:extensible": true,
+  "definitions": {
+    "asset": {
+      "properties": {
+        "xdm:repository": {
+          "title": "repository",
+          "type": "string",
+          "description": "repository of the asset.",
+          "enum": ["aem", "web"],
+          "meta:enum": {
+            "aem": "AEM",
+            "web": "Web"
+          }
+        },
+        "xdm:type": {
+          "title": "type",
+          "type": "string",
+          "description": "type of the asset.",
+          "enum": ["image", "html", "json"],
+          "meta:enum": {
+            "image": "Image",
+            "html": "HTML",
+            "json": "Json"
+          }
+        },
+        "xdm:sourceURL": {
+          "title": "sourceURL",
+          "type": "string",
+          "description": "source URL of the asset. eg) URL of an image asset http://cdn.com/myimage.png"
+        },
+        "xdm:destinationURL": {
+          "title": "destinationURL",
+          "type": "string",
+          "description": "certain assets like images can have a destination, when clicked upon. eg) http://ecommerce.com/product"
+        }
+      },
+      "required": ["xdm:repository", "xdm:sourceURL"]
```

Comment:
@usankara , let's put the required in the field group where you intend to use it. This data type could be used elsewhere too.

Reply:
@mjasrotia , ok , sure. do we need to change the datatype name to "asset" instead of "decisioning-asset" and move to a common directory at higher level (say https://github.com/adobe/xdm/tree/master/components/datatypes) ..?

Reply:
asset in general could be very generic and can be interpreted in a million ways, document as an asset, service as an asset , application as an asset and so on. To me , it deserves its own class with an assetID, name and then corresponidng field groups defining different assets. Let's keep this one here for your use cases :)